lib/common/http1client.c

```diff
@@ -363,6 +366,8 @@ static void on_head(h2o_socket_t *sock, const char *err)
     h2o_buffer_consume(&sock->input, client->bytes_to_consume);
     client->bytes_to_consume = 0;
     client->sock->bytes_read = client->sock->input->size;
+    if (reader == on_req_chunked)
+        client->_seen_at_least_one_chunk = 0;
```

Comment:
Do we need to initialize `_seen_at_least_one_chunk` here?

It is my understanding that all the members of `h2o_http1client_t` (with the exception of `super`) is zero-cleared when initialized in `setup_client`.

Assuming that, I might prefer setting this here to avoid confusion.

Reply:
Yes, good call, we don't need the initialization here, fixing